interface/src/app/about/about.tpl.html

```diff
@@ -1,8 +1,21 @@
-<div>
-  <h1 class="page-header">
-    About ADAGE
-  </h1>
-  <p>
-    This is where some explanatory text about ADAGE should go.
-  </p>
+<div class="page-header">
+  <h1>About ADAGE</h1>
 </div>
+<h3>
+  <p>
+    The ADAGE webserver allows users without programming expertise to analyze
+    signatures extracted from large compendia of gene expression data via
+    unsupervised machine learning methods. We primarily designed the server to
+    support models constructed by
+    <a href="http://msystems.asm.org/content/1/1/e00025-15">ADAGE</a>
+    and
+    <a href="http://biorxiv.org/content/early/2017/04/10/078659">eADAGE</a>;
+    however, the server was designed to be general and can be used for other
+    types of machine learning models as well.
```

Comment:
Please include a closing `</p>` tag as well. There are some circumstances when it is allowed to be omitted, but it is much easier to write correct HTML by not doing that.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p